src/w3c/templates/cgbg-sotd.js

```diff
@@ -39,7 +39,9 @@ ${!conf.sotdAfterWGinfo ? [conf.additionalContent] : ""}
       href='${`https://lists.w3.org/Archives/Public/${conf.wgPublicList}/`}'>archives</a>)${conf.subjectPrefix ? html`
     with <code>${conf.subjectPrefix}</code> at the start of your
     email's subject` : ""}.</p>
-  ` : ""}
+  ` : html`
+   <p>If you wish to make comments regarding this document, please raise them as <a href="${conf.issueBase}">GitHub issues</a>.</p>
```

Comment:
Note that this is conditional on `conf.issueBase` actually being there. Same below. A lot of specs won't have `conf.issueBase`, because they don't declare `respecConfig.github` (or manually set `conf.issueBase`).

Reply:
Oh. I thought `respecConfig.github` was a required config variable. 
Will fix this.